src/btree/bt_vrfy.c

```diff
@@ -638,7 +645,7 @@ __verify_tree(
                 continue;
             } else
                 WT_RET(ret);
-            ret = __verify_tree(session, child_ref, unpack, vs);
+            ret = __verify_tree(session, child_ref, unpack, child_ref->home->dsk->write_gen, vs);
```

Comment:
Is the write_gen always accessible here? i.e: is it possible that one of `child_ref`, `home`, or `dsk` are NULL?

Reply:
No. We already accessed the `child_ref->home->dsk->write_gen` a few lines before, so it is never be NULL here.